org.eclipse.scanning.test/src/org/eclipse/scanning/test/points/RasterTest.java

```diff
@@ -188,7 +188,7 @@ public void testFillingAMoreComplicatedBoundingRectangle() throws Exception {
 		assertEquals("Point list size should be correct", rows * cols, pointList.size());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "long". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)